cypress/integration/plugins/custom-import-map-dashboards/documentsLayer.spec.js

```diff
@@ -1,79 +0,0 @@
-/*
- * Copyright OpenSearch Contributors
- * SPDX-License-Identifier: Apache-2.0
- */
-
-import { BASE_PATH } from '../../../utils/constants';
-
-describe('Documents layer', () => {
```

Comment:
seems to be flaky due `with-security` enabled. Perhaps instead of removing we can do

something similar to [this](https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/cypress/integration/plugins/security-dashboards-plugin/aggregation_view.js#L27)

where we check 

```js
if (!Cypress.env('SECURITY_ENABLED')){
```
and wrap the entire test in it. So we can test it still without security and we should create a follow-up issue to look into it.

Reply:
Thanks, let me update!